src/configs/labels.js

```diff
@@ -483,6 +483,22 @@ const labels = {
       name: 'name',
       available: 'available'
     }
+  },
+  statusPatterns: {
+    select: [
+      { label: 'Так', value: 'true' },
+      { label: 'Ні', value: 'false' }
+    ]
+  },
+  materialPatterns: {
+    select: [
+      { label: 'Нитка для шиття', value: '6043b2ec3e06ad3edcdb7b17' },
+      { label: 'Мальмо', value: '6043a1f33e06ad3edcdb7b09' },
+      { label: 'Шкірзамінник', value: '6043aaab3e06ad3edcdb7b11' },
```

Comment:
Why you write material id value, we can delete or add new material and this information will be outdated. If you want to get info about the material you can get it from the redux store.

Reply:
Fixed